### PR TITLE
Add bits definition

### PR DESCRIPTION
**Pull request by skdltmxn**

### CODE REVIEW

File: src/um/bits.rs [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+use shared::minwindef::{BOOL, DWORD, FILETIME, ULONG};
+use um::unknwnbase::{IUnknown, IUnknownVtbl};
+use um::winnt::{HRESULT, LPCWSTR, LPWSTR, WCHAR};
+pub const BG_SIZE_UNKNOWN: UINT64 = -1i64 as u64;
+STRUCT!{struct BG_FILE_PROGRESS {
+    BytesTotal: UINT64,
+    BytesTransferred: UINT64,
+    Completed: BOOL,
+}}
+RIDL!(#[uuid(0x01b7bd23, 0xfb88, 0x4a77, 0x84, 0x90, 0x58, 0x91, 0xd3, 0xe4, 0x65, 0x3a)]
```
Comment (retep998) [inline src/um/bits.rs]: `RIDL` is an item level macro and should use `{}` instead of `()`.

File: src/um/bits10_1.rs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    fn FileRangesTransferred(
+        job: *mut IBackgroundCopyJob,
+        file: *mut IBackgroundCopyFile,
+        rangeCount: DWORD,
+        ranges: *const BG_FILE_RANGE,
+    ) -> HRESULT,
+});
+RIDL!(#[uuid(0xcf6784f7, 0xd677, 0x49fd, 0x93, 0x68, 0xcb, 0x47, 0xae, 0xe9, 0xd1, 0xad)]
+interface IBackgroundCopyFile6(IBackgroundCopyFile6Vtbl)
+    : IBackgroundCopyFile5(IBackgroundCopyFile5Vtbl) {
```
Comment (retep998) [inline src/um/bits10_1.rs]: `:` should be at the end of the previous line.

**Comment by skdltmxn** [comment]
~~It looks like AppVeyor is broken~~
Oh, it recovered itself somehow!

**Comment by retep998** [comment]
I restarted the appveyor build 😛

**Pull request merged**